### PR TITLE
"German additions"

### PR DESCRIPTION
**Pull request by switchhax**
I always wanted to say "I´m back". Well i finished high school now. In October i´ll start going to University.

Well now to "business". Another try to understand (or misunderstand) the guidelines for this project. I tried some new books, maybe you "like" them this time :)

### CODE REVIEW

File: free-science-books-de.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Fachrichtungen
+
+## Philologie
+* [Über die Dichtkunst beim Aristoteles](http://www.gutenberg.org/files/16880/16880-h/16880-h.htm)
+
+## Philosophie
+* [Betrachtungen über die Grundlagen der Philosophie](http://www.gutenberg.org/files/27532/27532-h/27532-h.htm)
+* [Leibnitz Monadologie](http://www.gutenberg.org/files/39441/39441-h/39441-h.htm)
+* [Tractatus Logico-Philosophicus](http://www.gutenberg.org/files/5740/5740-pdf.pdf?session_id=59a679d6eb04ae342b7d1e172efbe27cf6f576dc)
```
Comment (eshellman) [inline free-science-books-de.md]: need to leave out session id

File: free-science-books-de.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Fachrichtungen
+
+## Philologie
+* [Über die Dichtkunst beim Aristoteles](http://www.gutenberg.org/files/16880/16880-h/16880-h.htm)
```
Comment (eshellman) [inline free-science-books-de.md]: maybe better to use urls like http://gutenberg.org/ebooks/16880

**Comment by switchhax** [comment]
So before i rework this PR, i´ve got a few questions.
* So Project Gutenberg links are okay?
* If yes, they should be "named by their overview-page", right?
  Like you said `http://gutenberg.org/ebooks/16880`?

**Comment by eshellman** [comment]
PG links are OK. Might want to add year of original publication.
overview page is best, I think

**Comment by eshellman** [comment]
seems I forgot about this PR.

**Pull request merged**